.github/config.yml

```diff
@@ -0,0 +1,23 @@
+# https://github.com/behaviorbot/welcome
+# combines new-issue-welcome, new-pr-welcome, and first-pr-merge
+
+# Comment on first-time issues
+# https://github.com/behaviorbot/new-issue-welcome
+newIssueWelcomeComment: >
+  👋 Thanks for opening your first issue here! If you're reporting a bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
+
+  To make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/issues.md).
+
+# Comment on PRs from first-time contributors
+# https://github.com/behaviorbot/new-pr-welcome
+newPRWelcomeComment: >
+  💚 Thanks for opening this pull request! 💚
+
+  Please make sure to update documentation and tests when appropriate, and follow the [commit guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#commit-message-guidelines)
+
+  We get a lot of issues on this repo, so please be patient and we will review this as soon as we can.
+
+# Comment when a first-time contributor's PR is merged
+# https://github.com/behaviorbot/first-pr-merge
+firstPRMergeComment: >
+  Congratulations on merging your first pull request! 👍
```

Comment:
Will this be triggered even if we don't merge the GitHub way?

Reply:
No, this app purely uses GitHub Apps to listen on webhook events, like the PR being closed and then checks if the PR was merged and if it was that individual's first PR to be merged. So it depends on GitHub's `merged` property to be true for the PR.

Reply:
We often don't have that...